### PR TITLE
GS-299 : Fixed count and namespace queries

### PR DESCRIPTION
**Pull request by meriouma**

### CODE REVIEW

File: gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/service/visualizer/EntitiesServiceImpl.java [hunk 1]
```diff
@@ -31,19 +32,21 @@
...
 
 public class EntitiesServiceImpl implements EntitiesService {
     private final DatastoreHelper datastoreHelper;
     private final DefaultValueGenerator defaultValueGenerator;
+    private final DatastoreCountProvider countProvider;
 
     @Inject
     EntitiesServiceImpl(DatastoreHelper datastoreHelper,
-                        DefaultValueGenerator defaultValueGenerator) {
+                        DefaultValueGenerator defaultValueGenerator,
```
Comment (christiangoudreau) [inline gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/service/visualizer/EntitiesServiceImpl.java]: We should review that style convention to be eclipse friendly

File: gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/util/DatastoreCountProvider.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/**
```
Comment (christiangoudreau) [inline gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/util/DatastoreCountProvider.java:1]: Uhm, all headers should be moved to open source, I'll create a task

Comment (meriouma) [inline gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/util/DatastoreCountProvider.java:1]: I think there's already one created

File: gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/service/visualizer/EntitiesServiceImpl.java (hunk 1 above)
Comment (meriouma) [inline gae-studio/gae-studio-server/src/main/java/com/arcbees/gaestudio/server/service/visualizer/EntitiesServiceImpl.java]: Yea I was planning to add checkstyle and fix the style after those 2 PRs

**Comment by Pacane** [comment]
Nice! I didn't know if we could get the stats with both production and dev environements, but this is good with me! LGTM

**Pull request merged**